notesWindow.py

```diff
@@ -0,0 +1,118 @@
+from PyQt5.QtWidgets import QApplication,QWidget,QLabel,QTextEdit,QVBoxLayout,QGridLayout,QPushButton,QLineEdit,QSizePolicy,QScrollArea,QMessageBox
```

Comment:
this is a part of the note making feature. It should not be here i guess?

Reply:
I didn't create branches from master. I had create them one after another. That' why I think this has happened.

Reply:
I created them in order. That's why they are like this.